LEGO1/lego/legoomni/src/infocenter/registrationbook.cpp

```diff
@@ -2,10 +2,26 @@
 
 #include "legoomni.h"
 
+#include "mxnotificationmanager.h"
+
 // STUB: LEGO1 0x10076d20
```

Comment:
This should not be a STUB.

Reply:
@Ramen2X wdym? I need the header for Register() function.